### DISCUSSION TITLE
v7 - TUnit

### DISCUSSION THREAD

**Discussion by thomhurst** [fluentassertions/fluentassertions#2862] [General]
Heya. I thought TUnit support would be in v7, but it doesn't seem to be. It's in the alpha release notes but not the official release ones.

Can also see it's not a registered framework when debugging: 
![image](https://github.com/user-attachments/assets/2e604a97-8588-4908-8e29-8097348d8c36)

Do you know when it'll be released?

**Comment by dennisdoomen** [comment]
Fluent Assertions 6 becomes 7, and 7 becomes 8....

See https://x.com/ddoomen/status/1863311674645856324
